taigit/src/reducers/taigaReducer.js

```diff
@@ -59,18 +66,73 @@ export const selectSprintProgressChartData = (state) => {
 }
 
 export const selectUserTaskDistributionChartData = (state) => {
+  var backgroundColor = [
+    'rgba(255, 99, 132, 0.6)',
+    'rgba(54, 162, 235, 0.6)',
+    'rgba(255, 206, 86, 0.6)',
+    'rgba(75, 192, 192, 0.6)',
+    'rgba(153, 102, 255, 0.6)',
+    'rgba(255, 159, 64, 0.6)',
+    'rgba(255, 99, 132, 0.6)'
+  ];
+  let taigaTaskStatsArr = state.taigaTaskStats;
+  let dictToProcessData = {};
+  // Deserialzation of received data from backend
+  for (let i = 0; i < taigaTaskStatsArr.length; i++) {
```

Comment:
It looks like this grabs data for the entirety of the project, this is great, and we can use this in the _Overview_ page. For the Taiga page, we're going to have the data display based on a sprint selected in the dropdown menu. Would you be able to add another selector function, that takes in a name or id of a sprint and then returns chart data for only that sprint?

Reply:
Sure, I can do that. Shall I create a new task to implement this with specific sprint or should just go with this one?